client/src/javascript/components/modals/settings-modal/DiskUsageTab.js

```diff
@@ -0,0 +1,134 @@
+import {Checkbox, Form, FormRow, Select, SelectItem, Radio} from 'flood-ui-kit';
+import {FormattedMessage, injectIntl} from 'react-intl';
+import React from 'react';
+
+import ModalFormSectionHeader from '../ModalFormSectionHeader';
+import SettingsStore from '../../../stores/SettingsStore';
+import SettingsTab from './SettingsTab';
+import SortableList from '../../general/SortableList';
+import DiskUsageStore from '../../../stores/DiskUsageStore';
+
+class DiskUsageTab extends SettingsTab {
+  tooltipRef = null;
+
+  state = {
+    diskItems: [],
+  };
+
+  componentWillMount() {
+    // assemble disk items from saved "mountPoints" and list of disks "disks"
+    const mountPoints = SettingsStore.getFloodSettings('mountPoints');
+    const disks = DiskUsageStore.getDiskUsage().reduce((a, c) => {
+      a[c.target] = c;
+      return a;
+    }, {});
```

Comment:
Let's rename variables `a` and `c` to something more meaningful. For example, `c` would be better represented as `disk` (or whatever it actually represents), and `a` would be better represented as `diskByTarget` or something like that.